woocommerce/woocommerce-ios
# Remove Inject wrappers around OrderDetailsViewController

## Description

I've noticed that in `trunk` Order Details doesn't show action button in navbar. Short investigation led me to some `Inject.ViewControllerHost(...)` wrappers added around `OrderDetailsViewController`. After removing them navbar seems to be fixed.

I don't know what is the exact reason (probably VCHost in the middle breaks NavigationController logic), but [docs](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/inject-hot-reload.md) say that Inject code shouldn't end up in production anyway, so I removed all the wrappers for both UIKit and SwiftUI.

## Testing

1. Go to the Orders tab, open any existing order.
2. Wait until order fully loads.
3. Check "•••" button in navbar. Tapping it should reveal "Edit" option.

## Screenshots

before | after
--|--
![Simulator Screen Shot - iPhone 13 - 2022-07-06 at 10 27 12](https://user-images.githubusercontent.com/3132438/177494205-09841edd-02de-4cbf-a681-f3b26cc5bbdc.png)|![Simulator Screen Shot - iPhone 13 - 2022-07-06 at 10 28 09](https://user-images.githubusercontent.com/3132438/177494218-456f1ad2-08c4-4227-8c15-ed3d914d59e5.png)


---
